### PR TITLE
Adding select fields for ids

### PR DESCRIPTION
**Pull request by PepiPetrov**
According to @kachar I replaced text fields for ID with select dropdowns.

### CODE REVIEW

**Comment by kachar** [comment]
@PepiPetrov There are some unrelated changes regarding `.husky` again, let's clear them so there's no change in that part

**Comment by PepiPetrov** [comment]
I can see it has only two files.

**Comment by kachar** [comment]
@PepiPetrov Looks like the mode has been changed for the executable files from 755 to 644

image

**Comment by PepiPetrov** [comment]
Strange. How can I fix it?

**Comment by kachar** [comment]
You might check this topic https://stackoverflow.com/a/10516406/668245

It makes git detect the changes in the file modes, which will help you see when they are changed

As for these changed files it will be easier if I join and help you clear them up

**Comment by PepiPetrov** [comment]
OK. I'll read it.

**Comment by kachar** [comment]
To restore the mode I've used

```
chmod 755 .husky/pre-commit manifests/postrelease.sh manifests/release.sh
```

**Comment by PepiPetrov** [comment]
Thank you for the info.

File: src/common/util/keycloak.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import getConfig from 'next/config'
```
Comment (kachar) [inline src/common/util/keycloak.ts:1]: @PepiPetrov This file has been moved to `frontend/src/middleware/auth/keycloak.ts` on `master`

There's no point in having two copies of the same file, so please remove this sone and keep only the one from `middleware/auth/keycloak`

**Review by kachar — APPROVED** [review]
@PepiPetrov Marvelous upgrade of the app. Thanks for your contribution.

Can you please split your contributions in smaller PRs that are providing dedicated functionality. 

Also some overview of the changes would be nice! The PR description now explains that person pickers will be developed, but the actual code gives more than that. It will make us lost if we want one part of the code, but not the other. QA team won't be aware of what they should test after such a PR

:v:

**Pull request merged**